en/docs/configurations.md

```diff
@@ -0,0 +1,24 @@
+# Configurations
+
+The extension offers a number of settings. These can be configured by modifying your [user or workspace settings](https://code.visualstudio.com/docs/getstarted/settings) or by modifying the` settings.json ` file of VS Code.
+
+| ID                                    | Description                                               |
+|---------------------------------------|-----------------------------------------------------------|
+| ballerina.codeLens.all.enabled        | Enable code lenses that appear in the editor view.           |
+| ballerina.enableConfigurableEditor    | Enable the configurable editor. This will open a  form to set values for [configurable variables](https://ballerina.io/learn/by-example/configurable-variables/) in the code when you [Run](../docs/build-and-run/build-and-run.md) the Ballerina code. |
```

Comment:
```suggestion
| ballerina.enableConfigurableEditor    | Enable the configurable editor. This will open a  form to set values for [configurable variables](https://ballerina.io/learn/by-example/configurable-variables/) in the code when you [run](../docs/build-and-run/build-and-run.md) the Ballerina code. |
```